packages/date-picker/src/DatePicker/DatePicker.tsx

```diff
@@ -33,13 +35,18 @@ export interface DatePickerProps
   /**
    * Accepts a DayOfWeek value to start the week on that day. By default,
    * it's set to Monday.
+   *  @default 1
```

Comment:
Does this show up in the docs? If not, I think it's fine to leave it out.
But if it's useful, I believe it should be `DayOfWeek.Mon`